build/config/PublishData.json

```diff
@@ -0,0 +1,36 @@
+{
+  "master": {
+    "nugetPush": [
```

Comment:
Is "allowedNuGetPushes" more sensible? This is not a list of things that is done automatically or manually _per se_, just an entry here means it's later allowed to do.